scripts/process/topo_county_boundaries.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash  
```

Comment:
(this is mostly Dave being curious...) Not following why we need a bash script to do this. -- oh!! upon a little googling, I see. The toposimplify implements the "Visvalingam’s" algorithm. Looks like that algorithm is available in the mapshaper package... https://github.com/ateucher/rmapshaper -- but I guess that topoquantize function is important too?

Reply:
Yes, and topo2geo and toposimplify and topoquantize are all said to be faster at the command line than via javascript (node.js), so i've been assuming that's also the case for R. also, having being converted to the D3 world, i don't see a big reason to prefer an R solution over Mike Bostock's widely used solution.

Reply:
👍